pkg/telemetery/settings.go

```diff
@@ -0,0 +1,13 @@
+package telemetery
+
+import (
+	"github.com/jaegertracing/jaeger/pkg/jtracer"
+	"go.opentelemetry.io/otel/metric"
+	"go.uber.org/zap"
+)
+
+type TelemeterySetting struct {
+	Logger        *zap.Logger
+	Tracer        *jtracer.JTracer
+	MeterProvider metric.MeterProvider
```

Comment:
why do we want metrics provider here? All Jaeger code operates on `metrics.Factory`

Reply:
Yes, noticed that. I initially had initialized tracer and meter as done in V2 telset, had changed the tracer to accept jTracer, wanted to study more before changing the metric. Because as I mentioned in the args metrics.Factory is not always being passed, sometimes the metricsQuerySrv is being passed as an arg.